src/main/sensors/acceleration.c

```diff
@@ -462,7 +462,7 @@ static void performAcclerationCalibration(void)
         if (!sensorCalibrationSolveForOffset(&calState, accTmp)) {
             accTmp[0] = 0.0f;
             accTmp[1] = 0.0f;
-            accTmp[1] = 0.0f;
+            accTmp[2] = 0.0f;
```

Comment:
Well spotted! I wonder if this is causing acc calibration errors in some cases

Reply:
Maybe?